extensions/wikia/CreateNewWiki/classes/BulkRevisionImporter.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+use Wikia\CreateNewWiki\Tasks\TaskContext;
+
+/**
+ * Import a set of pages into the wiki database in a bulk operation
```

Comment:
The MediaWiki provided importer class doesn't suit our needs—it makes way too many queries for the 200+ pages we need to import. So I created a class to support bulk imports.